asllib/doc/ASLTypingReference.tex

```diff
@@ -1158,16 +1504,16 @@ \section{TypingRule.Subtype\label{sec:TypingRule.Subtype}}
 \section{TypingRule.Supertype}
 
   \subsection{Prose}
-  \texttt{t} is a supertype of \texttt{s} if and only if \texttt{s} is a subtype of \texttt{t}.
+  $\vt$ is a supertype of $\vs$ if and only if $\vs$ is a subtype of $\vt$.
 
 
 \todocomment{
-ROMAN: This rule doesn't exist in the code. Also, it is redundant, since it just applies to the same arguments in reverse order. I recommend removing this section.
+ROMAN: This rule doesn't exist in the code. Also, it is redundant, since it just apply to the same arguments in reverse order. I recommend removing this section.
```

Comment:
applies

also, please remove if you think it is unnecessary - but make sure the term is not used anywhere